### PR TITLE
Pop in Chat Bubble Text

### PR DESCRIPTION
**Pull request by ThatDan123**
Chat bubble text now pops in!

CL: [Improvement] Chat bubble text now pops in!

-Clown is now even more annoying

-Settings menu contains some customisation over character pop in speed and time. Can also make it instant like the old chat

-Adds a singleton manager, will convert over all old managers in another PR

### CODE REVIEW

**Comment by corp-0** [comment]
I love the colourful clown text and me myself have no trouble reading it but I'm afraid it might hinder accessibility for some people. Could we maybe add a checkbox on the settings to enable and disable the effect?

File: UnityProject/Assets/Scripts/Managers/SettingsManager/DisplaySettings.cs [hunk 1]
```diff
@@ -77,6 +78,39 @@ public bool ChatBubbleSizeChanged
 				}
 			}
 			private bool chatBubbleSizeChanged = false;
+
+			public bool ChatButtleInstantChanged
+			{
+				get => chatButtleInstantChanged;
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Managers/SettingsManager/DisplaySettings.cs]: `chatButtle`? 🙂

**Review by NoooneyDude — DISMISSED** [review]

**Comment by NoooneyDude** [comment]
> I love the colourful clown text and me myself have no trouble reading it but I'm afraid it might hinder accessibility for some people. Could we maybe add a checkbox on the settings to enable and disable the effect?

Yep, but perhaps a generic "Accessibility" toggle, or similar, instead of something for this specific instance would be a good idea.

**Comment by ThatDan123** [comment]
> > I love the colourful clown text and me myself have no trouble reading it but I'm afraid it might hinder accessibility for some people. Could we maybe add a checkbox on the settings to enable and disable the effect?
> 
> Yep, but perhaps a generic "Accessibility" toggle, or similar, instead of something for this specific instance would be a good idea.

I've just done a setting for just the clown colour, think its best we have lots of options instead of just a generic accessibility toggle imo

**Review by Bod9001 — APPROVED** [review]

**Review by corp-0 — APPROVED** [review]

**Pull request merged**